src/networks.ts

```diff
@@ -82,6 +84,7 @@ export const PRODUCTION_NETWORKS: { [chainId: number]: PublicNetwork } = {
   },
   [CHAIN_IDs.BOBA]: {
     name: "Boba",
+    family: OP_STACK,
```

Comment:
Not 100% sure whether this is technically true, since Boba may not have followed OP through to Bedrock and beyond. It doesn't really matter in either case, since we don't actually use Boba, and we'd thoroughly test it again if we were to re-enable it.